### ISSUE TITLE
Feature request: compare sorted files

### ISSUE THREAD

**Issue by davidmoten** [davidmoten/big-sorter#5]
@deathmorlock
Extracted from #3

* Ability to compare 2 sorted files and save different entries to file
* Ability to compare 2 sorted files and save same entries to file
* Ability to compare 2 sorted files and delete duplicates from 1st file in 2nd

**Comment by davidmoten** [comment]
master now has support for the three requested features via these static methods:
* `Util.findSame`
* `Util.findDifferent`
* `Util.findComplement` 

See unit tests of new methods at https://github.com/davidmoten/big-sorter/blob/master/src/test/java/com/github/davidmoten/bigsorter/UtilTest.java.

Example:

```java
File a = ...
File b = ...
File c = ...
Comparator<String> comparator = Comparator.naturalOrder();
Serializer<String> ser = Serializer.linesUtf8();
try (
    Reader<String> readerA = ser.createReader(a); 
    Reader<String> readerB = ser.createReader(b);) {
        try (Writer<String> writerC = ser.createWriter(c)) {
            Util.findSame(readerA, readerB, comparator, writerC);
    }
}
```
After running, file c has the results.

**Comment by davidmoten** [comment]
released to Maven Central 0.1.13

**Comment by davidmoten** [comment]
master now has this improvement for files:

```java
// already sorted
File a = ...
// already sorted
File b = ...
// result of the operation
File out = ...
Util.findSame(a, b, Serializer.linesUtf8(), Comparator.naturalOrder(), out);
```

**Comment by davidmoten** [comment]
Released to Maven Central 0.1.15, closing as complete
